arrow-libs/gradle/libs.versions.toml

```diff
@@ -0,0 +1,59 @@
+[versions]
+animalSniffer = "1.5.0"
+arrowGradleConfig = "0.0.1-SNAPSHOT"
+autoService = "1.0-rc7"
+compileTesting = "0.18"
+coroutines = "1.5.2"
+dokka = "1.5.30"
```

Comment:
I'm not sure we can bump to `1.5.30` already.
The fixes we have in the jekyll module was build for `1.5.0`, if we want to bump the version we need to do manual testing of the website to verify nothing broke.

If a fix landed in 1.5.30, then we might be duplicating the fix. I.e. wrapping code blocks twice in ```lang

cc\\ @raulraja @franciscodr unless this version bump gets deployed only to the next version, then we can let this version bump go through and we an do manual testing on `next`. ?

Reply:
I downgraded it, no problem with that :)